infra/main.bicep

```diff
@@ -496,6 +507,16 @@ module storageRoleBackend 'core/security/role.bicep' = {
   }
 }
 
+module storageRoleSearchService 'core/security/role.bicep' = {
```

Comment:
This should only be added if they're using the integrated vectorization feature.